src/components/userProfile/userProfile.js

```diff
@@ -1,51 +1,84 @@
 import React from "react";
-import {getUserProfileInfo} from '../../services/userService';
+import {editUserInfo, getUserProfileInfo} from '../../services/userService';
 import Header from '../navigation/header';
-import {Button, Input, Layout} from 'antd';
+import {Button, Form, Input, Layout, Popconfirm} from 'antd';
 import {CheckOutlined, CloseOutlined, PlusCircleOutlined} from '@ant-design/icons';
 import NewCarModal from "../newCarModal/index";
+import {confirmMessage, errorMessage} from "../../services/alert.service";
+import {generalErrorMessages} from "../../constants/messages/general";
+import {inputValidationErrors} from "../../constants/messages/inputValidationErrors";
+import {userErrorMessages} from "../../constants/messages/user";
 
 /* now the class is using instead of function
  because it wasn't working with function,
  but it will be rewritten later
  */
 class UserProfilePage extends React.Component {
```

Comment:
Here it would be cool to use a functional component.
Class components remain 2016 😊.
( That's only my opinion )

Reply:
It won't be in this PR